skxray/speckle_visibility/speckle_visibility.py

```diff
@@ -0,0 +1,290 @@
+# ######################################################################
```

Comment:
Right, but one of the goals of this whole project is to reuse as much code between techniques as possible.  Putting general use code in to domain specific files (two levels deep) works against that goal.